src/ChatTextToSpeech.csproj

```diff
@@ -4,11 +4,10 @@
     <AzureFunctionsVersion>v2</AzureFunctionsVersion>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.CognitiveServices.ContentModerator" Version="2.0.0" />
-    <PackageReference Include="Microsoft.Azure.CognitiveServices.Language.TextAnalytics" Version="2.8.0-preview" />
+    <PackageReference Include="Azure.AI.TextAnalytics" Version="1.0.0-preview.4" />
```

Comment:
Update to GA.

Reply:
We just released v5.0.0 https://www.nuget.org/packages/Azure.AI.TextAnalytics so it will be great if we could use that one

Reply:
updated